README.md

```diff
@@ -50,3 +50,60 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Add development build stage to dockerfile
+ docker build --target development --tag module5:dev .
+
+ ## Running the App in dev environment
+
+(base) RaviDommatas-Mac-mini:DevOps-Course-Starter rkdommata$ docker run -p 5000:5000 --env-file .env module5
```

Comment:
Its a bit weird to include your full prompt `RaviDommatas-Mac-mini:DevOps-Course-Starter rkdommata`